config/config-autoscaler.yaml

```diff
@@ -57,6 +57,13 @@ data:
     # horizontally scale the application based on this target concurrency.
     container-concurrency-target-default: "100"
 
+    # The requests per second (RPS) target default is what the Autoscaler will
+    # try to maintain when the Revision specifies unlimited RPS.
+    # Even when specifying unlimited RPS, the autoscaler will
+    # horizontally scale the application based on this target RPS.
+    # Must be greater than 1.0.
```

Comment:
A comment on mutual exclusivity on the metric might be good now that we have more than just one?

Reply:
Done.